autopush/src/main.rs

```diff
@@ -61,3 +60,17 @@ fn main() -> Result<()> {
     signal.recv().unwrap();
     server.stop().chain_err(|| "Failed to shutdown properly")
 }
+
+/// Create a new channel subscribed to the given signals
```

Comment:
So, if I understand, `notify` starts a rust thread to monitor for POSIX signals, and then direct them back to the main thread via a `crossbeam_channel` buffered for 100 messages? (To replace the deprecated `chan_signal` library.)

I'm curious why the buffer is so large, particularly for 2 kill signals.

Reply:
why not have it be large? memory is cheaper than impossible to reproduce runtime errors.

Reply:
It's 100 because I copied the recommended impl. from https://crates.io/crates/chan-signal and it's the same buffer size `chan_signal::notify` used.

It could be lower for our use case but it's not terribly important